### DISCUSSION TITLE
No spoilers

### DISCUSSION THREAD

**Discussion by wombats17** [riffnshred/nhl-led-scoreboard#336] [General]
My wife and I usually watch the games after the fact and I was wondering is there a way to have a no spoiler option?  I like the game banner and seeing when my team plays and against whom just do not want to see the score or current or past games.  I don't mind trying to program it myself I just thought I was ask before that.  Thank you!

**Comment by riffnshred** [comment]
This is not an option in this repository, although It might be possible to tweak it to do so. You can forget about team summaryt thought, the available data  for each team on the API is current, but I do believe  tho that each game data contains the teams record at that time the game was over, so you could use that.
